apisix/discovery/nacos/init.lua

```diff
@@ -315,6 +315,7 @@ local function fetch_full_registry(premature)
                 host = host.ip,
                 port = host.port,
                 weight = host.weight or default_weight,
+                metadata = host.metadata or {},
```

Comment:
Why the default value is an empty table, is nil more appropriate?

Reply:
In fact, nacos currently returns an empty json object when metadata is not configured, which corresponds to an empty table in lua. In order to keep this feature stable, this is a failback mechanism. Prevent future destructive changes to existing apisix code if metadata does not return or null is returned by nacos next release